### PR TITLE
Add supoort for HEAD and allow_redirects

### LINKED ISSUE #51: Add an option to disable redirects in "http" check command.
For the check defined like

```python
def check():
  status_code = http('https://service.dns.name/file/very-large-file.zip').code()
  return {"status_code": status_code}
```

I would like to have `status_code` = 302 when the server returns redirect. The reason is that I only need to check that the `very-large-file.zip` is accessible, but I don't want to download this file in ZMON.

Currently this check returns `status_code` = 200, which means that ZMON follow redirects. Would be good to somehow disable such behaviour.

### PR DESCRIPTION
**Pull request by mohabusama**
Fixes #51

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/http.py [hunk 1]
```diff
@@ -157,6 +161,12 @@ def __init__(
         self.verify = verify
         self._headers = headers or {}
         self.oauth2 = oauth2
+        self.method = method.lower()
```
Comment (hjacobs) [inline zmon_worker_monitor/builtins/plugins/http.py]: this allows users to overwrite the method with arbitrary content, either make it have two underscores or move the check into `__request__`

**Comment by hjacobs** [comment]
Can you also update the documentation?

File: zmon_worker_monitor/builtins/plugins/http.py (hunk 1 above)
Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/http.py]: Done.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**